### DISCUSSION TITLE
In the KL operating system, there will be garbled code. But when I use other browsers and on Windows, there won't be garbled code.

### DISCUSSION THREAD

**Discussion by wosiyourfather** [cefsharp/CefSharp#4580] [Q&A]
In the KL operating system, there 
![image](https://github.com/cefsharp/CefSharp/assets/53590491/ecbdc404-3519-47eb-9010-cfb8898f8a7b)
will be garbled code. But when I use other browsers and on Windows, there won't be garbled code.
![1693386638623](https://github.com/cefsharp/CefSharp/assets/53590491/8822dbeb-cf2d-4e25-8fc2-14ce8ae84bc6)

**Comment by amaitland** [comment]
> KL operating system

What is this exactly? `Windows` is the only supported `OS`.

Check the log file for errors https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#log-file

Make sure you distributed the relevant `.pak` files.

See also https://github.com/cefsharp/CefSharp/blob/master/CONTRIBUTING.md#cefsharp-vs-chromium-embedded-frameworkcef

Try the `cefclient` sample application.

**Reply by wosiyourfather** [reply]
is ：KYLIN。

**Reply by wosiyourfather** [reply]
PAK files are available. I tried to add the log and there was indeed an error. Here are a few, take a look
![1693445302960](https://github.com/cefsharp/CefSharp/assets/53590491/13906290-b0dd-48eb-9c37-463e1950f8df)

**Reply by amaitland** [reply]
`CefSharp` only supports running on `Windows` as it's written using `Visual C++`.

The `upstream` `Chromium Embedded Framework (CEF)` project supports `C/C++` and can run on `Windows/Linux/Mac`.

https://bitbucket.org/chromiumembedded/cef/wiki/Tutorial.md
